bazel/external/quiche.BUILD

```diff
@@ -1248,6 +1272,7 @@ envoy_cc_test_library(
     repository = "@envoy",
     tags = ["nofips"],
     deps = [
+        ":quiche_common_platform_mutex",
```

Comment:
Why does :quic_platform_test depend on :quiche_common_platform_mutex?  (It doesn't in Google3.)

Reply:
Removed. I thought I needed it to, but clearly not.